### PR TITLE
hooks scripts

### PR DESCRIPTION
**Pull request by vijaykanthm28**

### CODE REVIEW

**Comment by indykish** [comment]
@vinomca-megam  Your review comments..

File: scripts/hooks.rb [hunk 1]
```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/ruby
+
+ONE_LOCATION=ENV["ONE_LOCATION"]
+
+if !ONE_LOCATION
+    RUBY_LIB_LOCATION="/usr/lib/one/ruby"
```
Comment (vinomca-megam) [inline scripts/hooks.rb]: If we dont use the RUBY_LIB_LOCATION please remove

File: scripts/hooks.rb [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    VMDIR="/var/lib/one"
```
Comment (vinomca-megam) [inline scripts/hooks.rb]: remove this one also

File: support/post_vertice_sync.rb [hunk 3]
```diff
@@ -118,8 +94,8 @@ def exit_error
 begin
```
Comment (indykish) [inline support/post_vertice_sync.rb]: @vijaykanthm28  Add a log("Hook - vertice #{status} #{status} started")

File: support/post_vertice_sync.rb [hunk 4]
```diff
@@ -118,8 +94,8 @@ def exit_error
...
    m  = Megam::Assembly.update(KEYS)
 rescue Exception => e
-log_error e.to_s
-exit -1
+  log_error e.to_s
+  exit -1
 end
 
 log "Hook finished:"
```
Comment (indykish) [inline support/post_vertice_sync.rb]: @vijaykanthm28 Add a log("Hook - vertice #{status} #{status} finished")

File: support/post_vertice_sync.rb [hunk 5]
```diff
@@ -59,45 +55,25 @@ def exit_error
...
+TEMPLATE = ARGV[0]
+
+if TEMPLATE.nil?
+    log_error("Exiting due to Template is  nil.")
+    exit -1
+end
 
-TEMPLATE = ARGV[1]
 content = TEMPLATE
 s  = Base64.decode64(content.inspect)
```
Comment (indykish) [inline support/post_vertice_sync.rb]: @vijaykanthm28  Remove the content variable/s.

File: support/post_vertice_sync.rb [hunk 6]
```diff
@@ -59,45 +55,25 @@ def exit_error
...
```
Comment (indykish) [inline support/post_vertice_sync.rb]: @vijaykanthm28  If the file doesn't exists (CONFIG_FILE) or the required parameters (STATE/STATUS, TEMPLATE) are blank then exit with an error.

File: support/post_vertice_sync.rb [hunk 7]
```diff
@@ -59,45 +55,25 @@ def exit_error
...
 
-HOST_ID = ARGV[0]
-
-if HOST_ID.nil?
-    exit -1
-end
+STATE  = ARGV[1]
+STATUS = ARGV[2]
 
 @doc  = Nokogiri::XML(s)
```
Comment (indykish) [inline support/post_vertice_sync.rb]: @vijaykanthm28  Replace `s` with `Base64.decode64(TEMPLATE)`

File: support/post_vertice_sync.rb [hunk 8]
```diff
@@ -3,13 +3,9 @@
 ONE_LOCATION=ENV["ONE_LOCATION"]
```
Comment (indykish) [inline support/post_vertice_sync.rb]: @vijaykanthm28  Add our banner at the top.

**Comment by vijaykanthm28** [comment]
@vinomca-megam please review

**Comment by indykish** [comment]
@vinomca-megam  Thanks for your review, i have deleted the ones i had which were dups of yours.

**Pull request merged**